torch/_torch_docs.py

```diff
@@ -3502,6 +3502,36 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.ge`.
 """)
 
+add_docstr(torch.gradient,
+           r"""
+gradient(input, *, spacing=None, dim=None, edge_order=1) -> List of Tensors
+
+This function is analogous to NumPy's gradient function.
+
+Args:
+    {input}
+
+Keyword args:
+    spacing (scalar, list of scalar, list of Tensor, optional): represents either positions of
+        points that underlying functions values provided, or spacing between coordinate points of them.
+    dim (int, list of int, optional): represents directions that we are interested in
+        finding gradients.
+    edge_order (int, optional): Expected to be 1 or 2.
```

Comment:
unsupported (must be equal to its default value)

Reply:
done